video/video_library_interface_review.rb

```diff
@@ -0,0 +1,184 @@
+# DO NOT MERGE - this file for comments only
+
+def analyze_labels_gcs path:
+  # [START analyze_labels_gcs]
+  # path = "Path to a video file on Google Cloud Storage: gs://bucket/video.mp4"
+
+  require "google/cloud/video_intelligence/v1beta1"
+
+  video_client = Google::Cloud::VideoIntelligence::V1beta1::VideoIntelligenceServiceClient.new
+  features     = [:LABEL_DETECTION]
```

Comment:
OOC: will lower-case symbols work today, eg. would `:label_detection` work?

Reply:
Not currently. This would be a straightforward feature to add in GAX, though.